pkgs/tools/networking/vegeta/default.nix

```diff
@@ -5,17 +5,17 @@
 
 buildGoModule rec {
   pname = "vegeta";
-  version = "12.11.0";
+  version = "12.11.1";
   rev = "e04d9c0df8177e8633bff4afe7b39c2f3a9e7dea";
```

Comment:
What's this rev for?

Reply:
I this was there from previous commits. I used `nix-update` to update it and it seems this rev is not used.
Removing.
Thank you for the review.